clusterloader2/docs/create-new-test.md

```diff
@@ -0,0 +1,67 @@
+# Create a new performance test
+
+## Select and copy the baseline test job config
+
+A great majority of our tests lie at the [test-infra] repository. All the
+configs there are written as [ProwJobs]. There are two types of tests being used
+there:
+
+-   **periodic** which is a test that runs at given cron interval
+-   **presubmit** which may be used as a presubmit for GitHub pull requests
+
+And there are three scales at which those are run: 100, 500 and 5000 nodes. Once
+it's known what is to be achieved it's good to start with a test that is the
+closest to it. Here are a few good example starting points, all running
+[load-test][]:
+
+Presubmits:
+
+-   100 nodes: [pull-kubernetes-e2e-gce-100-performance]
+-   5000 nodes: [pull-kubernetes-e2e-gce-scale-performance-manual]
+
+Periodics:
+
+-   100 nodes: [ci-kubernetes-e2e-gci-gce-scalability]
+-   5000 nodes: [ci-kubernetes-e2e-gce-scale-performance]
+
+We propose to start with 100 node tests as they are less costly and much
+quicker, shortening debugging cycle and when those tests pass, switch to 5k node
```

Comment:
Changed to:

``` markdown
After that it's possible to switch to 5k
node scale but since running such a test is quite expensive we recommend using
[ci-kubernetes-kubemark-gce-scale] as the baseline instead ([kubemark] is a tool
used for simulating a cluster).       
```